<launch>
    <!-- Arguments -->
    <arg name="rviz" default="true" />

    <!-- Start RViz -->
    <node type="rviz" name="rviz" pkg="rviz" args="-d $(find fetch_delivery_system)/config/rviz/segmentation.rviz" if="$(arg rviz)"/>

    <!-- Start MoveIt -->
    <include file="$(find fetch_delivery_system)/launch/include/moveit.launch" />

    <!-- Start the object segmentation node -->
    <node type="segment_object" name="segment_object" pkg="fetch_delivery_system" />
</launch>